README.md

```diff
@@ -188,11 +188,10 @@ This list is solely a compilation of apps that adopt the Material You design gui
 <sub>[📜Table Of Contents](#-table-of-contents)</sub>
 
 ---
-## 📱Known apps
+## 📱 Known apps
 ### 👥 Social Media Clients
 - **Reddit** <br></br>
 *How to make Reddit clients work after **API changes?*** Checkout this [Guide](https://www.reddit.com/r/redditsync/comments/14lkhnz/this_revanced_patch_makes_sync_work_after_api/)!
-	- `MDY` [Sync for Reddit](https://play.google.com/store/apps/details?id=com.laurencedawson.reddit_sync)
```

Comment:
Why is sync removed? i get its not on the play store anymore, but cant you link to [the apk mirror page?](https://www.apkmirror.com/apk/red-apps-ltd/sync-for-reddit/sync-for-reddit-v23-05-03-1718-release/)

Reply:
> Why is sync removed? i get its not on the play store anymore, but cant you link to [the apk mirror page?](https://www.apkmirror.com/apk/red-apps-ltd/sync-for-reddit/sync-for-reddit-v23-05-03-1718-release/)

Yes, but it will not working, because dev made this app paid as I know. I will add new link, when it will possible.

Reply:
No? It works when you follow my guide. (2 lines above!) I just tested it!